lib/linguist/generated.rb

```diff
@@ -386,6 +387,13 @@ def npm_shrinkwrap_or_package_lock?
       name.match(/npm-shrinkwrap\.json/) || name.match(/package-lock\.json/)
     end
 
+    # Internal: Is the blob a generated Yarn Plug'n'Play?
+    #
+    # Returns true or false.
+    def generated_yarn_plugnplay?
+      name.match(/(^|\/)\.pnp\.(c|m)?js$/)
```

Comment:
This method is going to return [a `MatchData` object](https://ruby-doc.org/core-2.5.1/Regexp.html). Please add `!!` to return a boolean.

Reply:
Okay, done :+1: was copying 1 of the 2 other methods that didn't use `!!` - Refactored them too.